proto/viam/app/v1/app.proto

```diff
@@ -816,9 +816,9 @@ enum FragmentErrorType {
   FRAGMENT_ERROR_TYPE_CYCLE_DETECTED = 4;
 }
 
-message FragmentWithError {
+message ResolvedFragment {
```

Comment:
dont have a strong opinion on terminology but floating some synonyms to resolved to make sure we feel good about it: "expanded" or "flattened". i still like "resolved" but figured id toss some out to see how we feel

Reply:
ooo thank you for the suggestions, i think i still prefer "resolved" because it leaves us some space in case computing fragments means something else in the future